packages/@ember/canary-features/index.ts

```diff
@@ -16,7 +16,7 @@ export const DEFAULT_FEATURES = {
   EMBER_LIBRARIES_ISREGISTERED: null,
   EMBER_IMPROVED_INSTRUMENTATION: null,
   EMBER_NAMED_BLOCKS: null,
-  EMBER_GLIMMER_SET_COMPONENT_TEMPLATE: true,
+  EMBER_GLIMMER_SET_COMPONENT_TEMPLATE: false,
```

Comment:
Same for here, we actually want to remove this line, and replace anything that uses it to have truthy semantics. You can start by replacing any references to it by with literally just the literal `true`, but eventually you'd want to simplify the code.

So, for instance... 

```js
import { EMBER_GLIMMER_SET_COMPONENT_TEMPLATE } from '@ember/canary-features';

// ...

let someThing;

if (EMBER_GLIMMER_SET_COMPONENT_TEMPLATE) {
  someThing = () => {
    doThisWhenFeatureIsEnabled();
  };
} else {
  someThing = () => {
    doThisWhenFeatureIsDisabled();
  };
}
```

becomes 

```js
// don't need the import anymore

// ...

let someThing;

if (true /* treat the flag as always enabled */) {
  someThing = () => {
    doThisWhenFeatureIsEnabled();
  };
} else {
  someThing = () => {
    doThisWhenFeatureIsDisabled();
  };
}
```

then further simplifying it...

```js
// don't need the import anymore

// ...


function someThing() {
  doThisWhenFeatureIsEnabled();
}

// the "feature disabled branch can be removed 
```

Reply:
sorry for the confusion.  I am new to open source.